### PR TITLE
Create ParaSpell_follow-up2.md

### PR DESCRIPTION
**Pull request by dudo50**
#### Project Abstract

This is a follow-up grant for the previous 2 we have submitted with ParaSpell:
- Phase 1: https://github.com/w3f/Grants-Program/pull/1118
- Phase 2: https://github.com/w3f/Grants-Program/pull/1245

This proposal makes SDK more efficient, replaces outdated localhost network library in UI from Parachain-Launch into Zombienet & includes support for new XCM compatible Parachains / Update Parachains included with new support they added recently.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by Noc2 — APPROVED** [review]
Thanks a lot for the application. I'm happy to approve it and mark it as ready for review. But not sure if I already mentioned this before, I think the tool itself should be funded via the treasury in the near future. So I would suggest applying for treasury funding for maintaining the tool and adding additional features after this grant.

**Comment by dudo50** [comment]
@Noc2 thank you for your suggestions and approval. We have applied for treasury funding already as you suggested, but we were told, that we need to have some funding from treasury pior to maintenance grant. We plan on creating small spending proposal there also in the meanwhile (while completing this proposal if it gets approved) so we will be ready to go for maintenance proposal that you suggested within 2 months. 

With kind regards,
Team ParaSpell

File: applications/ParaSpell_follow-up2.md
Comment (keeganquigley) [inline applications/ParaSpell_follow-up2.md:207]:
```suggestion
- **Costs:** 10,000 USD 💰️
```

File: applications/ParaSpell_follow-up2.md
Comment (keeganquigley) [inline applications/ParaSpell_follow-up2.md:201]:
```suggestion
- **Total Costs:** 10,000 USD
```

File: applications/ParaSpell_follow-up2.md
Comment (keeganquigley) [inline applications/ParaSpell_follow-up2.md:216]:
```suggestion
| 1. | Merge currencyId & currency in XCM Transfers | Merge currency ID and currency symbol into one so user do not need to enther both. Raised in issue: https://github.com/paraspell/xcm-sdk/issues/16 |
```

**Comment by keeganquigley** [comment]
@dudo50 thanks for the application. I'm happy to approve you for this one, and then as David suggested, I would also suggest the treasury as a route for further funding. Thanks for your continued work on this!

**Comment by dudo50** [comment]
@keeganquigley, thanks for your continuous trust in our work. I surely am open to having a follow-up call with you as Kusama is running on an OpenGov system with which we do not have experience yet. Thanks for your offer and feel free to reach out to me about this I will be very happy. dudo.morhac@gmail.com

With kind regards,
Team ParaSpell

**Comment by dudo50** [comment]
@Noc2 , @keeganquigley committing suggested changes might have removed your approvals. Sorry for this. I thought, that I should commit them as I had that option showing.

With kind regards,
Team ParaSpell

**Comment by keeganquigley** [comment]
> @Noc2 , @keeganquigley committing suggested changes might have removed your approvals. Sorry for this. I thought, that I should commit them as I had that option showing. Edit: All three suggested changes are now committed.
> 
> With kind regards, Team ParaSpell

thanks @dudo50 for making the updates. No worries, @Noc2 will re-approve when he sees it. Thanks, I will reach out via email to schedule the call!

**Review by keeganquigley — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/introducing-complex-set-of-xcm-tools-paraspell/5625/1

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/enhancing-cross-chain-experience-of-your-dapps-with-revolutionary-xcm-tool-set-from-paraspell/8792/1

**Pull request merged**
